package.json

```diff
@@ -18,19 +18,22 @@
     "prepublish": "npm run build"
   },
   "dependencies": {
+    "adm-zip": "0.4.13",
```

Comment:
We already have archiver and jszip as dependencies. Any reason to pull in a 3rd zip library?

Reply:
I think `archiver` only creates zips not extracts them.

`jszip` didn't have an easy way to take a path of a zip and extract it so I chose adm which we use in other repos. Docs I looked at were here https://stuk.github.io/jszip/documentation/howto/read_zip.html.

Can take another look at setting it up if you think it's worthwhile.

Reply:
We already do it here: https://github.com/microsoft/tfs-cli/blob/86123e37d047f3d2cbbcb15a1bff3c52d3049806/app/exec/extension/init.ts

Reply:
Thanks! Wasn't aware. I will extract the code and reuse it.